docs/core/tools/dotnet-test.md

```diff
@@ -441,14 +441,18 @@ dotnet test -h|--help
 With Microsoft Testing Platform, `dotnet test` operates faster than with VSTest. The test-related arguments are no longer fixed, as they are tied to the registered extensions in the test project(s). Moreover, MTP supports a globbing filter when running tests. For more information, see [Microsoft.Testing.Platform](../testing/microsoft-testing-platform-intro.md).
 
 > [!WARNING]
-> `dotnet test` doesn't run in environments that have test projects using both VSTest and Microsoft Testing Platform in the same solution, as the two platforms are mutually incompatible.
+> When opting-in Microsoft.Testing.Platform via `dotnet.config`,  `dotnet test` expects all test projects to use Microsoft.Testing.Platform. It is an error if any of the test project uses VSTest.
 
 #### Implicit restore
 
 [!INCLUDE[dotnet restore note](~/includes/dotnet-restore-note.md)]
 
 #### Options
 
+> [!NOTE]
+> You can use only one of the following options at a time: `--project`, `--solution`, `--directory`, or `--test-modules`. These options can't be combined.
+> In addition, when using `--test-modules`, you cannot specify `--arch`, `--configuration`, `--framework`, `--os`, or `--runtime`. These options are not relevant for an already-built module.
```

Comment:
This is WIP in https://github.com/dotnet/sdk/pull/47616. @rainersigwald @baronfel I need a bit of confirmation on the behavior before merging this docs PR please.